utils/rapids-colab.sh

```diff
@@ -13,10 +13,26 @@ echo "will BREAK"
 echo " "
 echo "Please enter in the box your desired RAPIDS version (ex: '0.10' or '0.11', between 0.9 to 0.11, without the quotes) and hit Enter. "
 read RAPIDS_VERSION
-if [ $RAPIDS_VERSION == "0.11" ] ;then
-  echo "Please COMPARE the \"SCRIPT TO COPY\" with the code in the above cell.  If they are the same, just type any key.  If not, do steps 2-4." 
+MULT="100"
+
+#conditions accidental inputs
+RAPIDS_RESULT=$(awk '{print $1*$2}' <<<"${RAPIDS_VERSION} ${MULT}")
```

Comment:
What is MULT for? I couldn't find any other reference to it. Are you using it as an argument in another script?

Reply:
it lets the script check the version as an integer for some of the `>=` statements for a few of the install cases that we have.